tools/ka-customizer-runner

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash -ex
 
+BRANCH=$1
```

Comment:
is this script supposed to be run by another script or humans? if humans, then we should probably add some usage message?